### PR TITLE
Multilingual support constants added-by Suvarna

### PR DESCRIPTION
**Pull request by suvarnakale**
Multilingual support constants added-by Suvarna

### CODE REVIEW

File: src/app/admin/dlogin/dlogin.component.html [hunk 1]
```diff
@@ -4,56 +4,56 @@
...
 
 
         <div class="d-flex flex-row row mb-3">
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">Facility Center</p>
-                <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">RECORD</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">View/Add</a>
+                <p class="p16 mb-1 font-600">{{'FACILITY_CENTER' | translate}}</p>
+                <h2 class="font-600">{{'0' | translate}}</h2>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html]: @suvarnakale 0 is value. Not requred here, please remove

File: src/app/admin/dlogin/dlogin.component.html [hunk 2]
```diff
@@ -4,56 +4,56 @@
...
+                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
             </div>
 
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">Doctors</p>
-                <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">RECORD</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">View/Add</a>
+                <p class="p16 mb-1 font-600">{{'DOCTORS | translate'}}</p>
+                <h2 class="font-600">{{'0' | translate}}</h2>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html]: @suvarnakale remove here too - 0

File: src/app/admin/dlogin/dlogin.component.html [hunk 3]
```diff
@@ -4,56 +4,56 @@
...
+                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
             </div>
 
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">Beneficiary</p>
-                <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">RECORD</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">View/Add</a>
+                <p class="p16 mb-1 font-600">{{'BENEFICIARY' | translate}}</p>
+                <h2 class="font-600">{{'0' | translate}}</h2>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html]: @suvarnakale here too

File: src/app/admin/ownership/ownership.component.html [hunk 4]
```diff
@@ -80,11 +79,11 @@ <h3>Coming Soon</h3>
             <div class="ml-2">
                 <span class="m-2 mr-3 p14">
                     <input type="checkbox" id="vehicle1" name="vehicle1" value="Bike" unchecked />
-                    <label for="vehicle1" class="mr-2 p14"> &nbsp; Mobile Number</label>
+                    <label for="vehicle1" class="mr-2 p14"> &nbsp; {{'MOBILE_NUMBER' | translate}}</label>
 
                     <select name="visibility" id="visibility" class="p14 w-185 p-2 fieldInput">
                         <option value="public"> <i class="fa fa-ellipsis-h" aria-hidden="true"></i>
-                            educationboard.email
+                            {{'EDUCATIONBOARD_EMAIL' | translate}}
```
Comment (Pratikshakhandagale) [inline src/app/admin/ownership/ownership.component.html]: @suvarnakale here we are showing value or its just label? please confirm once ` educationboard.email`

File: src/app/admin/ownership/ownership.component.html [hunk 5]
```diff
@@ -126,11 +125,11 @@ <h3>Coming Soon</h3>
                 <div class="ml-2">
                     <span class="m-2 mr-3 p14">
                         <input type="checkbox" id="vehicle1" name="vehicle1" value="Bike" unchecked />
-                        <label for="vehicle1" class="mr-2 p14"> &nbsp; Mobile Number</label>
+                        <label for="vehicle1" class="mr-2 p14"> &nbsp; {{'MOBILE_NUMBER' | translate}}</label>
     
                         <select name="visibility" id="visibility" class="p14 w-185 p-2 fieldInput">
                             <option value="public"> <i class="fa fa-ellipsis-h" aria-hidden="true"></i>
-                                educationboard.email
+                                {{'EDUCATIONBOARD_EMAIL' | translate}}
```
Comment (Pratikshakhandagale) [inline src/app/admin/ownership/ownership.component.html]: @suvarnakale here we are showing value or its just label? please confirm once `educationboard.email`

File: src/app/admin/publish/publish.component.html [hunk 6]
```diff
@@ -2,17 +2,17 @@
   <div class="wrapped mp-0" *ngIf="!isShow">
     <div class="card" style="width: 450px; height: 299px">
       <div class="card-body" style="margin: 10%; padding: 5%">
-        <p class="wrapped bTitle">Publish to Explore APIs</p>
+        <p class="wrapped bTitle">{{PUBLISH_TO_EXPLORE_APIS | translate}}</p>
```
Comment (Pratikshakhandagale) [inline src/app/admin/publish/publish.component.html]: @suvarnakale single quote('  ') is missing here

File: src/app/admin/dlogin/dlogin.component.html (hunk 3 above)
Comment (suvarnakale) [reply src/app/admin/dlogin/dlogin.component.html]: okay

**Pull request closed without merging**